app/controllers/categories_controller.rb

```diff
@@ -0,0 +1,45 @@
+class CategoriesController < ApplicationController
+  before_action :find_category, only: [:show, :edit, :update]
```

Comment:
Think of the way to prevent url hacks. If I add in url ``https://wetsy.herokuapp.com/categories/700`` category which doesn't exist it should display some error message. Right now it just shows that the requested page does not exist